worlds/doom_1993/docs/setup_en.md

```diff
@@ -11,9 +11,9 @@
 
 ## Installing AP Doom
 1. Download [APDOOM.zip](https://github.com/Daivuk/apdoom/releases) and extract it.
-2. Copy DOOM.WAD from your steam install into the extracted folder.
+2. Copy DOOM.WAD from your steam installation into the extracted folder.
    You can find the folder in steam by finding the game in your library,
-   right clicking it and choosing *Manage→Browse Local Files*.
+   right-clicking it and choosing `Manage → Browse Local Files`.
```

Comment:
Code blocks should be reserved for referencing code, filenames, or other explicit system information. See my above usage of a code block to indicate an explicit filename. Menu items were correctly emphasized with bold previously.